Services/Masterdata/Services.Masterdata.EF/Services.Masterdata.EF.csproj

```diff
@@ -1,8 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+  <ItemGroup>
+    <None Remove="RawSql\StoredProcedures\sp_GetMasterdataRecursiveTree.sql" />
+  </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="..\..\..\Core\Data.EF.SqlServer\Core.Data.EF.SqlServer\Core.Data.EF.SqlServer.csproj" />
     <ProjectReference Include="..\Services.Masterdata\Services.Masterdata.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Migrations\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="RawSql\StoredProcedures\sp_GetMasterdataRecursiveTree.sql">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
```

Comment:
add here also the new stored procedure: sp_GetMasterdataFilterByRelatedMasterdata.sql